component/src/main/java/org/wso2/extension/siddhi/io/email/source/EmailSource.java

```diff
@@ -54,116 +54,117 @@
+        description = "The `Email` source allows you to receive events via emails. An `Email` source can be " +
+                "configured using the `imap` or `pop3` server to receive events. This allows you to filter the" +
+                " messages that satisfy the criteria specified under the `search term` option. The email source " +
+                "parameters can be defined in either the `deployment yaml` file or the stream definition. The email" +
+                " sink first checks the stream definition for parameters. If no parameters are configured there, the" +
+                " email source checks the `deployment.yaml` file. If the parameter configurations are not available " +
+                "in either place, the default values are considered (i.e., if they are available). If you need to " +
+                "configure server system parameters that are not provided as options in the stream definition, they " +
+                "need to be defined in the `deployment yaml` file under email source properties. For more " +
+                "information about `imap` and `pop3` server system parameters, see the following.\n" +
+                "[JavaMail Reference Implementation - IMAP Store](https://javaee.github.io/javamail/IMAP-Store)\n" +
+                "[JavaMail Reference Implementation - POP3 Store Store](https://javaee.github.io/javamail/POP3-Store)",
         parameters = {
         @Parameter(name = "username",
-                description = "user name of the email account. (e.g. wso2mail is the username of wso2mail@gmail.com"
-                        + "mail account.",
+                description = "The user name of the email account. e.g., `wso2mail` is the username of the" +
+                        " `wso2mail@gmail.com` mail account.",
                 type = { DataType.STRING}),
         @Parameter(name = "password",
-                description = "password of the email account",
+                description = "The password of the email account.",
                 type = { DataType.STRING }),
         @Parameter(name = "store",
-                description = "Store type that used to receive emails. it can be either imap or pop3.",
+                description = "The store type that used to receive emails. Possible values are `imap` and `pop3`.",
                 type = {DataType.STRING },
                 optional = true,
                 defaultValue = "imap"),
         @Parameter(name = "host",
-                description = "Host name of the server "
-                        + "(e.g. host name for a gmail account with imap store : 'imap.gmail.com'). "
-                        + "The default value imap.gmail.com' is only valid if email account is a gmail account"
-                        + " with imap enable.",
+                description = "The host name of the server. The email address should always be " +
+                        "`<USERNAME>@<HOST_NAME>.com'. (e.g., If `gmail` is the host name, the " +
```

Comment:
@<HOST_NAME>.com is not correct. As an example if host name for gmail account for imap server is imap.gmail.com

Reply:
Deleted my fork due to some errors. Can I do this correction via a fresh pull request?